src/main/java/core/basesyntax/strategy/OperationHandler.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.FruitTransaction;
+
+public interface OperationHandler {
+    void executeOperation(FruitTransaction fruitTransaction);
```

Comment:
OperationHandler -> handleOperation
OperationExecutor -> executeOperation
Be consistent in the naming)